packages/opentelemetry/lib/index.js

```diff
@@ -0,0 +1,89 @@
+const { diag, DiagLogLevel } = require('@opentelemetry/api');
+const {
+	getNodeAutoInstrumentations
+} = require('@opentelemetry/auto-instrumentations-node');
+const { Resource } = require('@opentelemetry/resources');
+const opentelemetrySDK = require('@opentelemetry/sdk-node');
+const {
+	SemanticResourceAttributes
+} = require('@opentelemetry/semantic-conventions');
+const appInfo = require('@dotcom-reliability-kit/app-info');
+const {
+	OTLPTraceExporter
+} = require('@opentelemetry/exporter-trace-otlp-proto');
+const { NoopSpanProcessor } = require('@opentelemetry/sdk-trace-base');
+const logger = require('@dotcom-reliability-kit/logger');
+// OpenTelemetry Traces Endpoint to which to export traces (e.g. http:localhost:4318/v1/traces).
+/**
+ * Set up OpenTelemetry tracing.
+ *
+ * @param {object} options
+ *      Options passed into the function parameter.
+ * @param {string | null} [options.tracesEndpoint]
+ *      The URL to send OpenTelemetry trace segments to, for example http://localhost:4318/v1/traces.
+ */
+function opentelemetry({ tracesEndpoint }) {
```

Comment:
**suggestion:** not a blocker but maybe the entire options object should be optional too? This would match other packages [like crash handler](https://github.com/Financial-Times/dotcom-reliability-kit/blob/b41508441601f51a3dec4d0ad0e0d79a119c6f28/packages/crash-handler/lib/index.js#L21) and it means that if you don't want to specify any options you can call the function without having to pass in an empty object:

```suggestion
/**
 * Set up OpenTelemetry tracing.
 *
 * @param {object} [options]
 *      Options passed into the function parameter.
 * @param {string | null} [options.tracesEndpoint]
 *      The URL to send OpenTelemetry trace segments to, for example http://localhost:4318/v1/traces.
 */
function opentelemetry({ tracesEndpoint } = {}) {
```

Reply:
Yeah this flexibility makes sense